src/charts/trends/trends-chart.directive.js

```diff
@@ -114,7 +139,34 @@ angular.module('patternfly.charts').directive('pfTrendsChart',
         showYAxis: '=?'
       },
       replace: true,
-      templateUrl: 'charts/trends/trends-chart.html'
+      templateUrl: 'charts/trends/trends-chart.html',
+      controller: ['$scope',
+        function ($scope) {
+          $scope.getPercentageValue = function () {
+            return Math.round($scope.getLatestValue() / $scope.chartData.total * 100.0);
+          };
+          $scope.getLatestValue = function () {
+            return $scope.chartData.yData[$scope.chartData.yData.length - 1];
+          };
+          $scope.getChartHeight = function () {
+            var retValue = 60;
```

Comment:
When I try:
controller: function ($scope) {
        const SMALL = 30, LARGE = 60;
...
I get:
src/charts/trends/trends-chart.directive.js
  144:10  error  Unexpected token const
✖ 1 problem (1 error, 0 warnings)
Warning: Task "eslint:target" failed. Use --force to continue.
Aborted due to warnings.